docs/index.html

```diff
@@ -7,13 +7,94 @@
     <title>TinDog</title>
     <link rel="stylesheet" href="./styles/styles.css">
     <script src="https://code.jquery.com/jquery-3.6.0.js" integrity="sha256-H+K7U5CnXl1h5ywQfKtSj8PCmoN9aaq30gDh27Xc0jk=" crossorigin="anonymous"></script>
-    <style>#sidenav {display: none;}</style>
+    <style>#sidenav {display: none;}
```

Comment:
```
    <style>#sidenav {display: none;}
```
This line is super important - On page load, it makes sure that the dropdown menu is closed